src/main/java/sysc4806/graduateAdmissions/model/Term.java

```diff
@@ -0,0 +1,45 @@
+package sysc4806.graduateAdmissions.model;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * Represents a Term (4 month period)
+ * Used to Store Deadlines and denote which Term is active
+ *
+ * @author Eric
+ */
+@Setter
+@Getter
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+@Entity
+public class Term {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private boolean active;
+    private String deadline;
+    private Season season;
+    private String year;
+
+    /**
+     * Creates a Term with an autogenerated id and given parameters
+     *
+     * @param deadline String representing the deadline for this term
+     * @param season Season determining the season of the term
+     * @param year String representing the year of the term
+     * @param active boolean showing if the term is active or not
+     */
+    public Term(String deadline, Season season, String year, boolean active){
```

Comment:
Possibly also need a starting date for when you can begin applying to the term

Reply:
You can only apply to a term when its active and only one term (should) be active, so i think having a starting date would be redundant. if there's another reason for it that i didn't think of I could add it

Reply:
But what would you use to determine whether a term is active or not? I guess you can have it so when one term closes, the next one automatically opens, but this also may not always be the case for the time windows.

Reply:
the admin role will be the the one to open the term for applications (ie. set active to true via a button probably). 

There's no reason that only one term should be active at a time though. Fall2020 and Fall2021 applications shouldn't both be open at the same time, but I should be able to apply for either Summer2020 of Fall2020 right now

Reply:
If we wanted to add something that automatically activates a term, we would need a starting date. but doing that kind of automation is probably not necessary for this project. People can start applying to a term when the terms "active" boolean is true, and the Admin will have to decide when a term opens manually.